Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3449,7 +3449,7 @@ def qradar_search_retrieve_events_command(client: Client,
     Returns:
         CommandResults: The results of the command.
     """
-    interval_in_secs = int(args.get('interval_in_seconds', 60))
+    interval_in_secs = int(args.get('interval_in_seconds', 30))
```

Comment:
Let's represent this value via the `yml` default value, so it'll be transparent to the user.